Resources/doc/index.rst

```diff
@@ -311,11 +311,14 @@ of the ``knp_menu_get`` function:
 More Advanced Stuff
 -------------------
 
-* :doc:`Menus as Services <menu_service>`
-* :doc:`Custom Menu Renderer <custom_renderer>`
-* :doc:`Custom Menu Provider <custom_provider>`
-* :doc:`I18n for your menu labels <i18n>`
-* :doc:`Using events to allow extending the menu <events>`
+.. toctree::
+    :maxdepth: 1
+
+    menu_service
+    custom_renderer
+    custom_provider
+    i18n
+    events
```

Comment:
this will use the title of the linked pages. i think that is a good idea, but some of the titles will change. just saying - i am +1 on the change.